MaintenanceManager/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.0.7] - 2022-10-29
+### Added
```

Comment:
Pls change label to Changed instead of Added. Added is used to indicate any client facing API changes